amivapi/events/emails.py

```diff
@@ -59,6 +61,91 @@ def send_confirmmail_to_unregistered_users_bulk(items):
         send_confirmmail_to_unregistered_users(item)
 
 
+@schedulable
+def remindermail(item):
+    """Send a reminder email to all participants registered to an event
+
+    Args:
+        item: The item, which was just inserted into the database
+    """
+    event = item
+
+    # Fetch all the infos needed for the content
+    if event['title_en'] is not "":
+        title = event['title_en']
+    else:
+        title = event['title_de']
+
+    if event['location'] is not "":
+        location = event['location']
+    else:
+        location = ""
+
+    if event['time_start'] is not None:
+        date_time_event = event['time_start'].strftime(' %d %B %Y at %-H:%M ')
+    else:
+        date_time_event = " NaN "
+
+    fields = {
+        'location': location,
+        'datetime': date_time_event,
+        'title': title
+    }
+
+    # Populate content text with fetched infos
+    email_content = current_app.config['REMINDER_EMAIL_TEXT'] % fields
```

Comment:
As fair as I can see, `REMINDER_EMAIL_TEXT` is not in the current app settings. I think we should provide a default, similar to the signup confirmation